core/isolate.rs

```diff
@@ -389,6 +389,11 @@ impl Isolate {
     self.check_last_exception()
   }
 
+  fn run_microtasks(&self) -> Result<(), ErrBox> {
+    unsafe { libdeno::deno_run_microtasks(self.libdeno_isolate) };
+    self.check_last_exception()
```

Comment:
Probably you can leave out the check_last_exception? Maybe we can leave that to the caller for performance reasons.

Reply:
Removed.